mypy/parse.py

```diff
@@ -18,6 +18,11 @@ def parse(source: Union[str, bytes],
     The python_version (major, minor) option determines the Python syntax variant.
     """
     is_stub_file = fnam.endswith('.pyi')
+    if options.transform_source is not None:
+        if isinstance(source, str):  # Work around mypy issue
+            source = options.transform_source(source)
+        else:
+            source = options.transform_source(source)
```

Comment:
Is this because of https://github.com/python/mypy/issues/1533? It looks like this should not be needed with the `Any` above.

Reply:
Yeah, it's caused by that issue, and this is not actually needed.